Camunda.Api.Client.Tests/Camunda.Api.Client.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
This was done because netcoreapp2.0 does not support system.runtime.compilerservices.unsafe. The tests fail on my machine, but they didn't pass beforehand, so I didn't put too much of a push into it.